### PR TITLE
CYOA V1

### PR DESCRIPTION
**Pull request by mcdaniel67**
I have finished V1 of my CYOA. I put it in it's own repo, which can be found here: 
https://github.com/mcdaniel67/spotigraph

It is a related artists graph generator using the Spotify API. All relations are generated by Spotify. There are some screenshots of the finished product in the repo.

### CODE REVIEW

**Comment by djkotowski** [comment]
Any chance you could merge that repo with this one?  We can't easily monitor history on exercise submissions outside this repo.

**Comment by mcdaniel67** [comment]
Is this what you were looking for?

**Comment by djkotowski** [comment]
Thanks.

File: spotigraph/lib/graph_json.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Converts our graph data into the JSON required by vis.js
+class GraphJSON
+  def to_json(graph, depth)
+    @depth = depth
+    return nodes_to_json(graph), edges_to_json(graph)
```
Comment (mcdaniel67) [inline spotigraph/lib/graph_json.rb]: Is it possible to return two values from a function without using return? Would it just look like `[nodes_to_json(graph), edges_to_json(graph)]` ?

**Pull request closed without merging**